### PR TITLE
Support import comment for MySQL, PostgreSQL and MSSQL

### PR DESCRIPTION
**Pull request by nguyenalter**
## Summary
- Support import comment:
    - MySQL: convert `CREATE TABLE ... COMMENT = '...';` to table note
    - PostgreSQL: Use `COMMENT ON "table_name" IS '...';` or `COMMENT ON "schema_name"."table_name" IS '...';` to create/override table note
    - MSSQL: Use `EXEC sp_addextendedproperty ...` to add notes for table/column
- Example:
    - PostgreSQL: 
         ``` sql
        CREATE TABLE "schemaA"."locations" (
          "id" int PRIMARY KEY,
          "name" varchar
        );
        COMMENT ON TABLE "schemaA"."locations" IS 'This is a note in table "locations"';
        ```
     - Convert to DBML: 
          ```
          Table "schemaA"."locations" {
            "id" int [pk]
            "name" varchar
            Note: 'This is a note in table "locations"'
          }
          ```
## Issue
- https://github.com/holistics/dbml/issues/40
- https://github.com/holistics/dbml/issues/224

## Lasting Changes (Technical)
- Update `DbmlExporter`: export to DBML code now will write table note inside `Table ... {}` instead of table settings position
- Update MySQL, PostgreSQL and MSSQL parsers to support import table/column comment

## Checklist

Please check directly on the box once each of these are done

- [ ] Documentation (if necessary)
- [x] Tests (integration test/unit test)
- [ ] Integration Tests Passed
- [ ] Code Review

### CODE REVIEW

File: packages/dbml-core/src/parse/mysql/parser.pegjs [hunk 1]
```diff
@@ -84,10 +84,18 @@ TableSyntax
...
-      : {...table_name, fields}
+  let res = {...table_name, fields};
+
+	if (options && options.comment) {
+		// convert CREATE TABLE ... COMMENT '...' to Table notes
+		const note = options.comment;
+		res = { ...res, note }
+	}
+	if (indexes) {
+		res = { ...res, indexes }
```
Comment (NQPhuc) [inline packages/dbml-core/src/parse/mysql/parser.pegjs:95]:
```suggestion
		res.indexes  = indexes
```

File: packages/dbml-core/src/parse/mysql/parser.pegjs [hunk 2]
```diff
@@ -84,10 +84,18 @@ TableSyntax
...
-    // return statement
-    return indexes ? 
-    	{...table_name, fields, indexes} 
-      : {...table_name, fields}
+  let res = {...table_name, fields};
+
+	if (options && options.comment) {
+		// convert CREATE TABLE ... COMMENT '...' to Table notes
+		const note = options.comment;
+		res = { ...res, note }
```
Comment (NQPhuc) [inline packages/dbml-core/src/parse/mysql/parser.pegjs:92]: There's no need to copy res into itself here, it simply wastes memory
```suggestion
		res.note = note
```

**Review by NQPhuc — COMMENTED** [review]
Lastly, should also update tests in dbml/cli nhe em. Some are failing.

File: packages/dbml-core/src/parse/postgresql/parser.pegjs [hunk 3]
```diff
@@ -104,18 +104,28 @@ parser = commands:command* {
 				break;
 			case "comment":
 				switch(syntax_name.toLowerCase()) {
-					case "column":
-						const table_comment = tables.find(table => table.name === value.relation_name);
-						const field_comment = table_comment.fields.find(field => field.name === value.column_name);
-						field_comment.note = value.text;
+					case "column": {
+						const { schemaName, tableName, columnName } = value;
+						const foundTable = tables.find(table => table.schemaName === schemaName && table.name === tableName);
```
[the remark this answers is not in the document]
Comment (nguyenalter) [reply packages/dbml-core/src/parse/postgresql/parser.pegjs]: done anh

File: packages/dbml-core/src/parse/mssql/statements/actions.js [hunk 4]
```diff
@@ -52,6 +52,28 @@ function handleEnums (_enum, ast) {
   field.type.args = _enum.values.map(value => `'${value.name}'`).join(', ');
 }
 
+function handleComment (_comment, ast) {
```
[the remark this answers is not in the document]
Comment (nguyenalter) [reply packages/dbml-core/src/parse/mssql/statements/actions.js]: removed underscore char

File: packages/dbml-core/src/parse/mssql/statements/actions.js [hunk 5]
```diff
@@ -52,6 +52,28 @@ function handleEnums (_enum, ast) {
...
+  if (_comment.type === 'table') handleTableNote(_comment, ast);
+  else if (_comment.type === 'column') handleFieldNote(_comment, ast);
+}
+
+function handleTableNote (_comment, ast) {
+  let schemaName = _comment.schemaName;
```
[the remark this answers is not in the document]
Comment (nguyenalter) [reply packages/dbml-core/src/parse/mssql/statements/actions.js]: fixed

File: packages/dbml-core/src/parse/mssql/statements/actions.js [hunk 6]
```diff
@@ -52,6 +52,28 @@ function handleEnums (_enum, ast) {
...
+  if (schemaName === 'dbo') schemaName = null; // treat dbo as public schema
+  const foundTable = findTable(ast, _comment.tableName, schemaName);
+  if (foundTable) foundTable.note = _comment.note;
+}
+
+function handleFieldNote (_comment, ast) {
+  let schemaName = _comment.schemaName;
```
[the remark this answers is not in the document]
Comment (nguyenalter) [reply packages/dbml-core/src/parse/mssql/statements/actions.js]: same as above

File: packages/dbml-core/src/parse/mysql/parser.pegjs (hunk 2 above)
Comment (nguyenalter) [reply packages/dbml-core/src/parse/mysql/parser.pegjs]: Refactored anh

File: packages/dbml-core/src/parse/mysql/parser.pegjs (hunk 1 above)
Comment (nguyenalter) [reply packages/dbml-core/src/parse/mysql/parser.pegjs]: Refactored too

**Review by NQPhuc — APPROVED** [review]
LGTM!

**Pull request merged**